Demo/src/main/java/com/braintreepayments/demo/PayPalFragment.java

```diff
@@ -57,7 +57,7 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         payPalClient = new PayPalClient(
                 requireContext(),
                 super.getAuthStringArg(),
-                Uri.parse("https://mobile-sdk-demo-site-838cead5d3ab.herokuapp.com/")
+                Uri.parse("https://mobile-sdk-demo-site-838cead5d3ab.herokuapp.com/braintree-payments")
```

Comment:
Updated to match with iOS https://github.com/braintree/braintree_ios/blob/94e22368d4aea7f1dd79f2c74177eff17afe5e21/Demo/Application/Features/PayPalWebCheckoutViewController.swift#L10